infra/base-images/base-runner/test_all.py

```diff
@@ -156,9 +158,51 @@ def use_different_out_dir():
       os.environ['OUT'] = initial_out
 
 
+def test_all_centipede(directory, allowed_broken_targets_percentage):
+  """Do bad_build_check on all fuzz targets of centipede."""
+  sanitizer = os.getenv('SANITIZER')
+
+  # For all projects:
```

Comment:
I'm confused why we are testing sanitized and unsanitized builds in the same build. Aren't these done completely separately?

Reply:
OK. I think I understand (though this is complicated). I think in helper.py you are doing both builds so that people can test locally. But I think in production you are keeping the builds seperate.
If i may suggest, maybe put the unsanitized build in a subdirectory, it will make it easier for people to repro crashes if their sanitized target is always in the same spot.